### DISCUSSION TITLE
is this the intended behavios when adding capturing `task with new project`

### DISCUSSION THREAD

**Discussion by molleweide** [nvim-neorg/neorg#490] [Q&A] [answered]
When capturing a task is the context or the project suppose to be top level?

This is what I get:
```norg 

* test new project
  #contexts vim
  - [ ] xxx xx arst
``` 

I was expecting this this but I am not sure now:
```norg
#contexts vim
* dummy
  - [ ] arst arst
```

This is a minor issue but I thought it useful to ask.

**Comment by danymat** [comment, marked as answer]
Hello, at the moment the contexts you create are relevant to the created task, and not the project. 
But this is a good improvement to let users choose the location of where they want to add the contexts

**Reply by max397574** [reply]
would context over project even work? (ast wise)

**Reply by danymat** [reply]
Yes, it would work as a context over a project will make all tasks inside the project share this context

**Reply by molleweide** [reply]
Allright, good to hear that it made sense. I was asking because I saw this as an example in the gtd docs.
My thinking was to group all of my vim related tasks under one context but I am also still learning gtd so I don't know what is the best way to do stuff.
